src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -90,6 +118,11 @@ class ProfileCallTreeContextMenu extends PureComponent {
           >
             Function Name
           </MenuItem>
+          {this.isJS()
```

Comment:
actually I don't understand why you don't just create a local `isJS` boolean variable, that you'd simply compute at the start of the `render` function.